### PR TITLE
READMEs: links, RTView documentaion.

### PR DESCRIPTION
**Pull request by denisshevchenko**
The purpose of this PR is to prepare this repository to the first release of RTView service.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,36 @@
 
 # cardano-benchmarking
 
-## preparation
+This repository contains two important tools for `cardano-node` benchmarking:
+
+1. `cardano-tx-generator`
```
Comment (CodiePP) [inline README.md]:
```suggestion
1. [cardano-tx-generator](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-tx-generator)
```

File: README.md [hunk 2]
```diff
@@ -2,21 +2,36 @@
...
+2. `cardano-rt-view-service`
```
Comment (CodiePP) [inline README.md]:
```suggestion
2. [cardano-rt-view-service](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-rt-view)
```

File: README.md [hunk 3]
```diff
@@ -2,21 +2,36 @@
...
+
+The first one is a generator of transactions, both for Byron and Shelley. For more details please [see the project](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-tx-generator).
+
+The second one is a service that allows to see the state of running `cardano-node` processes in the real time. For more details please [see the project](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-rt-view).
+
```
Comment (CodiePP) [inline README.md:13]: we need a small (250x180?) screenshot here..

File: README.md (hunk 2 above)
Comment (denisshevchenko) [reply README.md]: But I already provided a link to the project below.

File: README.md (hunk 3 above)
Comment (denisshevchenko) [reply README.md:13]: Yes, agree, we need a screenshot. But probably not here, but in `README.md` of `cardano-rt-view`?

File: README.md (hunk 2 above)
Comment (denisshevchenko) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (denisshevchenko) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (denisshevchenko) [reply README.md:13]: Added to `README.md` in `cardano-rt-view`.

**Review by CodiePP — APPROVED** [review]

**Pull request merged**